target/linux/bcm63xx/dts/bcm6318-tplink-td-w8968-v3.dts

```diff
@@ -0,0 +1,127 @@
+#include "bcm6318.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "TP-Link TD-W8968 v3";
+	compatible = "tplink,td-w8968-v3", "brcm,bcm6318";
+
+	aliases {
+		led-boot = &dsl_green;
```

Comment:
There's no good LED for this in the current definitions. The DSL light is not particularly useful (since DSL is not supported on this board), and it's next to the power LED, so it's an easy way to get this info out.